bot/exts/info/information.py

```diff
@@ -528,6 +543,9 @@ async def raw(self, ctx: Context, message: Message) -> None:
         """Shows information about the raw API response."""
         await self.send_raw_content(ctx, message)
 
+    async def _raw_context_menu_callback(self, interaction: discord.Interaction) -> None:
```

Comment:
```suggestion
    async def _raw_context_menu_callback(self, interaction: discord.Interaction, message: discord.Message) -> None:
```
Needs to accept a message argument in order to be classified as a message context menu

Reply:
This has already been fixed in a commit downstream. I was just writing a placeholder for the time.